jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerModule.java

```diff
@@ -0,0 +1,30 @@
+package org.secondbase.logging;
+
+import org.secondbase.core.SecondBase;
+import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.flags.Flag;
+
+public class JsonLoggerModule implements SecondBaseModule {
+
```

Comment:
I want to move these to the (now deleted) SecondBaseConfiguration. They apply to more than just logging. Consul registration, metrics appending, etc. The core module can start the logging module in the same way it starts the WebConsole. Maybe I can do that in another commit though.